static/Gruntfile.js

```diff
@@ -12,34 +12,30 @@ module.exports = function (grunt) {
     watch: {
       all: {
         files: ['**', '!**/node_modules/**', `!${outputDir}/**`],
-        tasks: ['jambobuild', 'webpack',],
-        options: {
-          spawn: false,
-        },
+        tasks: ['build-site']
       },
     },
   });
 
   grunt.loadNpmTasks('grunt-webpack');
   grunt.loadNpmTasks('grunt-contrib-watch');
+  grunt.registerTask('build-site', 'Builds the site.', buildSite);
+}
 
-  grunt.registerTask('jambobuild', 'Jambo build.',
-  function() {
-    // Force task into async mode and grab a handle to the "done" function.
-    var done = this.async();
-    // Run some sync stuff.
-    grunt.log.writeln('Processing task...');
-    // And some async stuff.
-    exec('npx jambo build', (error, stdout, stderr) => {
-      if (error) {
-        console.log(error.message);
-        done(false);
-        return;
-      }
-
-      stderr && console.error(stderr);
-      stdout && console.log(stdout);
-      done();
-    });
+/**
+ * Builds the site by spawning a jambo and webpack build
+ */
+function buildSite () {
+  const spawnedProcess = spawnSync('npx jambo build && npx webpack --config webpack-config.js', {
+    shell: true,
```

Comment:
is there a difference between using a shell and not?

Reply:
If shell is false, the first parameter is the parameter is the command and the second is an array of arguments. Also, I don't think the `&&` would work. With shell=true, we can easily run both the jambo build and the webpack build since the command will run within `/bin/sh`

Reply:
ok sgtm for the hotfix! I think for 1.23 we probably don't need to set shell to true though based on recent PR to add spawnSync to the gruntfile https://github.com/yext/answers-hitchhiker-theme/pull/872

Reply:
I imagine there's some tiny overhead for using a shell